src/components/styled/StyledWidthBox.js

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export const StyledWidthBox = styled.div`
+	max-width: 500;
```

Comment:
```suggestion
	max-width: 500px;
```

width requires a unit (e.g px)

Reply:
ohh youre absolutely right! I changed the value for testing things out and forgot to add px again.
I appreciate your recommendation, thank you!

Reply:
Check